pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -36,12 +39,26 @@ assert x11Support -> tcl != null
                   && xlibsWrapper != null
                   && libX11 != null;
 
+assert stdenv.lib.assertMsg (enableOptimizations -> (!stdenv.isDarwin))
```

Comment:
its actually clang that is the issue https://github.com/NixOS/nixpkgs/pull/103715